.github/workflows/format.yaml

```diff
@@ -0,0 +1,39 @@
+name: Format Codebase
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+
+    container: qmkfm/base_container
+
+    # protect against those who develop with their fork on master
+    if: github.repository == 'qmk/qmk_firmware'
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install dependencies
+      run: |
+        apt install dos2unix
+        pip3 install -r requirements.txt
+    - name: Format files
+      run: |
+        bin/qmk cformat
+        bin/qmk pyformat
+        bin/qmk fileformat
```

Comment:
Is it better to list multiple commands here, or would it be better to treat `qmk fileformat` as the entrypoint for this action and to put the `qmk cformat` and `qmk pyformat` calls into `qmk fileformat`?

Reply:
I was thinking adding something like `qmk format` would do the whole lot? But im sure there was something mentioned about cli commands calling other commands?

Reply:
I like the idea of `qmk format`, but not so much to keep up this PR any longer.